devops-challenges/code/q5/README.md

```diff
@@ -0,0 +1,23 @@
+## 5. Service roles in K8s
+
+```bash
+
+Scenario:
+
+You are required to complete the definition file located at `/home/ubuntu/tech-vault-q5/cluster-role-def.yml`. The file should have configurations to perform the following actions:
```

Comment:
The path to the question should be matching the folder structure so the reader does not get confused.